tests/cypress/tests-e2e/print.cy.js

```diff
@@ -0,0 +1,803 @@
+/// <reference types="cypress" />
+import { formatThousand } from '@/utils/numberUtils.js'
+
+const printID = 'print-123456789'
+
+describe('Testing print', () => {
+    beforeEach(() => {
+        cy.viewport(1920, 1080)
+
+        interceptCapabilities()
+        interceptShortLink()
+    })
+
+    function interceptCapabilities() {
+        cy.intercept('GET', '**/capabilities.json', (req) => {
+            req.reply({
+                body: {
+                    app: 'default',
+                    layouts: [
+                        {
+                            name: '1. A4 landscape',
+                            attributes: [
+                                {
+                                    name: 'name',
+                                    type: 'String',
+                                    default: 'this is a name',
```

Comment:
Would be much better to put this response mockup in a fixture files as here inline